client/src/tokens/typography.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { fontFamily, typeScale } from './typography';
+import { fontFamily, typeScale, exampleText } from './typography';
```

Comment:
I think we should use the `exampleText` in this file only, it is not something that is needed at the typography core tokens code but only in this storybook file.

Thanks - the data structure looks good, we just want to avoid extra 'stuff' in the core JS tokens file that is not actually needed to be shared across all code.

Reply:
It is my proposal: I moved the 'large body text' element up as the other elements are in descending order, for better consistency.
Before implementing it, I discussed it with Thibaus https://github.com/wagtail/wagtail/issues/8659 and received an approval.

Reply:
yep, so instead of creating and exporting from `client/src/tokens/typography.js` it would be inside this file `client/src/tokens/typography.stories.tsx`.

Something like...

```suggestion
import { fontFamily, typeScale } from './typography';

const exampleText = {
  'w-h1': 'Heading level (h1)',
  'w-h2': 'Heading level 2 (h2)',
  'w-h3': 'Heading level 3 (h3)',
  'w-h4': 'Heading level 4 (h4)',
  'w-label-1': 'Large label',
  'w-label-2': 'Medium label',
  'w-label-3': 'Small label',
  'w-body-text-large': 'Large body text',
  'w-body-text': 'Basic body text',
  'w-help-text': 'Help text',
};
```

Reply:
Got it, done.